app/build.gradle

```diff
@@ -38,6 +38,8 @@ spotless {
 }
 
 dependencies {
+    // Kotlin Libraries
+    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
```

Comment:
Why is this needed. It is 2 MB

Reply:
First of all, I see no use of this library, and even if it was used, I would have voted against it

Reply:
Alright sir. I added this because lint told me too

Reply:
Let me remove it